modules/jaeger-config-collector.adoc

```diff
@@ -21,19 +21,37 @@ The collectors are stateless and thus many instances of Jaeger Collector can be
 |Configuration options that define the Jaeger Collector.
 |
 
-|kafka:
-  producer:
-    topic: jaeger-spans
+|collector:
+  num-workers:
```

Comment:
This is still going to cause confusion, as this option and the next are defined under `collector/options`, e.g.
```
collector:
  options:
    collector:
      num-workers: 50
```
whereas the `replicas` field is:
```
collector:
  replicas: 5
```
I think we need two tables - one for the parameters that are used by the operator (or k8s), and the other for the options that are passed to the component (collector in this case).

Reply:
Let's see if I've got the distinction clear before I  start movings things around into new tables.
 - Parameters used by the operator are specified under the component in the YAML file (in this case under "collector")
- Parameters specified under "component: options{}"  are passed to the component

And is there somewhere where all this is written down?  I've been trying to piece things together from various example files in the Jaeger Operator repo, but a definitive list would be really useful.  The CLI flags (https://www.jaegertracing.io/docs/1.21/cli/#jaeger-collector-elasticsearch) don't really help in figuring out where the options would be nested in the configuration files.

Reply:
Correct, you have the distinction clear.

The CLI flags only appear in the component/options section (i.e. the second category), passed to the component.

Unfortunately the other types are not generated into easily understandable docs yet - there are some godocs: https://godoc.org/github.com/jaegertracing/jaeger-operator/pkg/apis/jaegertracing/v1

So for these, probably the examples are a good source still.

Reply:
> The CLI flags only appear in the component/options section (i.e. the second category), passed to the component.

Well, that is actually very helpful.  And thanks for the link to the godocs, I hadn't seen those before.

Reply:
@objectiser , this is somewhat of a stale PR.  I'm creating a new issue to deal with splitting the table format so that 
1 - we can move forward with removing this content from these branches and
2 - the tables get updated in the current and future branches.

JIRA is OSSMDOC-233.

Reply:
cc @jpkrohling